src/utils/report.js

```diff
@@ -52,11 +53,11 @@ export default function ({
     }
   }
 
-  const severity = (result.stylelint.ruleSeverities)
+  const actualSeverity = severity || (result.stylelint.ruleSeverities
     ? result.stylelint.ruleSeverities[ruleName]
```

Comment:
I actually did try setting this manually but it didn't work (presumably because it gets set implicitly via rule option)